monitor.go

```diff
@@ -63,7 +63,11 @@ type Monitor struct {
 func newMonitor(g reflect.Value, v specsp) (*Monitor, error) {
 	target := v["target"].(string)
 	val := v["val"].(string)
-	name := v["name"].(string)
+	label := v["label"].(string)
```

Comment:
The `name` serves as the widget's unique identifier throughout the game, while multiple widgets can have the same `label`.

I'll suggest to throw an error if `name` is not provided, or we should generate a unique value for it instead of using `label`.

Reply:
...  some old demos's config doesn't has that property "name",  and it works in spx 1.0

Reply:
🤣 We may generate some "unique value" for such cases, as using `label` as `value` may lead to naming conflicts.